README.md

```diff
@@ -41,3 +49,48 @@ import styles from './styles.scss'
 ```
 
 In order to better support usage inside Rails etc we have moved the public directory inside the client directory.  Thus the frameworks' root will not be cluttered with anything other than the client directory.  The generated app will run with `npm start` and you can still develop on `localhost:3000` if you are not in the context of a larger framework.
```

Comment:
I think `frameworks'` should be `framework's`? But I'm not entirely sure what you're referring to--the Rails (or other) framework you'd be generating into?